lib/cms/prayers.ex

```diff
@@ -19,11 +22,21 @@ defmodule CMS.Prayers do
 
   """
   def list_prayer_requests(%Scope{} = scope) do
+    user_group_ids =
+      from(g in Group,
+        join: gu in "groups_users",
+        on: g.id == gu.group_id,
+        where: gu.user_id == ^scope.user.id,
+        select: g.id
+      )
+      |> Repo.all()
+
```

Comment:
This is the same as lines 97 to 104 below. The prayers context should not need to now any details about user groups.

Reply:
One way to deal with this would be to preload at least the group ids when adding the user to the scope.